Bottom: b879bb68f09a79f76a5b59c736bc3753c31f111e
Top:    e73d0563209adf43ca0c6b7c7003ba71bd30c72c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-25 02:30:23 +0100

Refresh of try-to-make-gdk-and-gl-version

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 1550003..2b498b8 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -310,9 +310,6 @@ gint ghid_port_window_mouse_scroll_cb (GtkWidget * widget,
 				       GdkEventScroll * ev, GHidPort * out);
 
 
-gint ghid_port_drawing_area_expose_event_cb (GtkWidget * widget,
-					     GdkEventExpose * ev,
-					     GHidPort * out);
 gint ghid_port_drawing_area_configure_event_cb (GtkWidget * widget,
 						GdkEventConfigure * ev,
 						GHidPort * out);
